catroid/src/main/java/org/catrobat/catroid/content/bricks/UserBrick.java

```diff
@@ -210,11 +212,15 @@ public View getView(Context context, int brickId, BaseAdapter baseAdapter) {
 		view = BrickViewProvider.setAlphaOnView(view, alphaValue);
 
 		setCheckboxView(R.id.brick_user_checkbox);
-		onLayoutChanged(view);
+		onLayoutChanged(view, isUserBrickAdapter(baseAdapter));
 
 		return view;
 	}
 
+	private boolean isUserBrickAdapter(BaseAdapter baseAdapter) {
```

Comment:
Yes, I've changed it to be static.
Well, I personally don't like to pass any arguments with instanceof something operators, but prefer to wrap it into a function (that can also be reused later).